src/components/sbb-expansion-panel-header/readme.md

```diff
@@ -0,0 +1,71 @@
+The `sbb-expansion-panel-header` is a component which is meant to be used as a header in the `sbb-expansion-panel`, 
```

Comment:
My gut feeling would be to also structure this as you did in the accordion readme (i.e. separate into subtitles).
What do you think? Is it worth the effort?

Reply:
Yes, i did it in [23c4d25](https://github.com/lyne-design-system/lyne-components/pull/1860/commits/23c4d2557ee5eae7ed464f41ad211275b467056a);

Should we have it as a general pattern?

Reply:
I think so, yes. Maybe documenting it in CODING_STANDARDS?
Or what is your opinion on this?

Reply:
I totally miss this comment.. :|
I agree, I created a new draft issue in the "To do" column so we can update all the readmes and the CODING_STANDARD